docs/plus/pod.md

```diff
@@ -364,3 +364,86 @@ whether they are defined in the cdk8s app or not.
 
 > **Under the hood**: Co-location with managed pods will automatically
 > extract its labels and form the appropriate pod selector.
+
+## Connections
+
+Pod connections offer a simplified API to automatically create [network policies](./network-policy.md) on
+both ends of a connection. Accessing this API is done via the `connections` property
+of a specific `Pod`, which serves as one end of the connection.
+The other end is a network policy [peer](./network-policy.md#peers).
+
+### Allow To
+
+To allow connections from a `Pod` to a [peer](./network-policy.md#peers):
+
+```ts
+import * as k from 'cdk8s';
+import * as kplus from 'cdk8s-plus-22';
+
+const app = new k.App();
+const chart = new k.Chart(app, 'Chart');
+
+const redis = new kplus.Pod(chart, 'Redis', {
+  containers: [{ image: 'redis', port: 6379 }]
+});
+const web = new kplus.Pod(chart, 'Web', {
+  containers: [{ image: 'web' }]
+});
+
+web.connections.allowTo(redis);
+```
+
+This will allow the `web` pod to connect to the `redis` port on port 6379,
+and will allow the `redis` pod to accept connections from the `web` pod on port 6379.
+Note that the port is not specified in the `allowTo` invocation, it is automatically
+extracted from the `redis` pod definition.
+
+### Allow From
+
+To allow connections from a [peer](./network-policy.md#peers) to a `Pod`:
+
+```ts
+import * as k from 'cdk8s';
+import * as kplus from 'cdk8s-plus-22';
+
+const app = new k.App();
+const chart = new k.Chart(app, 'Chart');
+
+const redis = new kplus.Pod(chart, 'Redis', {
+  containers: [{ image: 'redis', port: 6379 }]
+});
+const web = new kplus.Pod(chart, 'Web', {
+  containers: [{ image: 'web' }]
+});
+
+redis.connections.allowFrom(web);
+```
+
+This will allow the `redis` pod to accept connection from the `web` pod on port 6379,
+and will allow the `web` pod to connect to the `redis` pod on port 6379.
+Note that the port is not specified in the `allowFrom` invocation, it is automatically
+extracted from the `redis` pod definition.
```

Comment:
What if the pod has multiple containers on different ports? Is there any way on allowFrom to specify ports myself if I want? (e.g. Maybe I'm calling `redis.connections.allowFrom` on a workload that has a sidecar that I also want to give peering access)

Reply:
Yes, you can pass ports to override the default. I'll add an example 👍